group_vars/wingedhelix_cluster/vars.yml

```diff
@@ -418,6 +418,18 @@ remote_users_in_local_groups:
         'umcg-vipt',
     ]
 #
+## Limit the storage for specific folder, by using a loop mount of formatted file
+#
+soft_partitions:
+  - path: /groups/umcg-lab/tmp07/sequencers
+    src: /root/mount.disk.img
+    fstype: xfs
+    size: 1           # in GB
+    user: sbsuser
+    group: umcg-lab
+    mode: 2750
+    hostname: wingedhelix
```

Comment:
This is not going to fly the way it is configured now:

1. `/root/mount.disk.img` is stored in the home of the root user, which by definition is on the local OS partition. May work for 1 GB, but will fail for larger disk images, which would fill the OS partition to 100%. 
2. When the disk image is mounted only on `wingedhelix` e.g. the UI, then on other machines the folder/path will not come from that disk image. As the folder for the sequencers is exported using the `smb_server `role from the SAI, the sequencers would not write into the loop mount on that machine. If you change `hostname: wingedhelix` into `hostname: "{{ groups['sys_admin_interface'] }}"`, the sequencers will write into the loop mounted image, but that is not mounted on the compute nodes that will process the data. On BB and CF `user_interface` == `sys_admin_interface` == `compute_node` as those are _all-in-one_ machines, but it would be nice if we can find a solution that works with a shared file system used on multiple machines...

Reply:
I believe some of this was already discussed at the standup. But here is additional info.
1. was just a short test role - the role worked on WH, but failed for `sbsuser`  - these values are now changed and machine to be deployed is `wh-sai`
2. the solution was for the SIAB machines, as those have the problems with limited storage size. The rest of the clusters have external storage (lustre, NFS) and for those we do not need soft partitions to limit the storage.
The `hostname` variable is just that: the machine hostname. It is not a variable for group of machines. I see no need to changing this and making it support as we have no usage for it at this stage.